doc/api/crypto.md

```diff
@@ -1706,6 +1706,71 @@ This should normally never take longer than a few milliseconds. The only time
 when generating the random bytes may conceivably block for a longer period of
 time is right after boot, when the whole system is still low on entropy.
 
+### crypto.randomFillSync(buf[, offset][, size])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `buf` {Buffer}
+* `offset` {number}
+* `size` {number}
+
+This function is similar to [`crypto.randomBytes()`][] but requires the first
+argument to be a [`Buffer`][] that will be filled as well as returned. Another
+significant difference between the two is that this function will always be
+synchronous. To use the asynchronous form, please see
+[`crypto.randomFill()`][]. The `offset` argument defaults to `0`.
+The `size` argument defaults to `buf.length - offset`.
+
+```js
+const buf = Buffer.alloc(10);
+crypto.randomFillSync(buf);
+console.log(buf.toString('hex'));
+
+crypto.randomFillSync(buf, 5);
+console.log(buf.toString('hex'));
+
+// The above is equivalent to the following:
+crypto.randomFillSync(buf, 5, 5);
+console.log(buf.toString('hex'));
+```
+
+### crypto.randomFill(buf[, offset][, size], callback)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `buf` {Buffer}
+* `offset` {number}
+* `size` {number}
+* `callback` {Function}
+
+This function is similar to [`crypto.randomBytes()`][] but requires the first
+argument to be a [`Buffer`][] that will be filled as well as returned. It also
+requires that a callback is passed in. The `offset` argument defaults to `0`.
```

Comment:
See above, use the param description to describe the params, removing the "The offset argument ... length - offset`" part from the description paragraph.